packages/cfd/src/Containers/cfd-dbvi-onboarding.tsx

```diff
@@ -91,12 +74,11 @@ const CFDDbViOnBoarding = ({
                     exit_classname='cfd-modal--custom-exit'
                 >
                     {showSubmittedModal ? (
```

Comment:
@amina-deriv i think we have 2 identical code blocks here: line 76 - 89 and line 100 - 113.
I suggest moving this block outside into a variable or a function in order to avoid code duplication🙂